src/attendee/Attendee.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+/**
+ * [[Attendee]] contains the information of an attendee.
+ */
+export default class Attendee {
```

Comment:
Yes I had `VideoSendingAttendee` to be more specific to this, since I got [this](https://github.com/aws/amazon-chime-sdk-js/pull/738#discussion_r500668649) comment from Andrew B on that, I made it more general. But yes if video only information needs to be sent later this would not be maintainable / applicable everywhere. 

What do you suggest here now? Rename it to more specific like `VideoSourceDetails` ?

Reply:
oh I missed that comment as GitHub collapsed it.

@aburnson could you review my comment on the Attendee class versus a simulcast-specific interface?

Reply:
If there are fields that are simulcast-specific, we can always extend this interface with another interface. Not now, but we should consider replacing the individual attendeeId/externalUserId with this Attendee interface in other APIs eventually.

Reply:
Extending an Attendee is a good idea. @devalevenkatesh You have two options.

```
interface VideoSendingAttendee extends Attendee
```

Or you can have an Attendee field, but this makes more sense for e.g., volume callback example
```
interface VideoSendingSource {
   attendee: Attendee
}
```

Reply:
RIght, the second approach is better for any changes later may it be related to Attendee or something simulcast specific etc. Thanks @simmkyu and @aburnson !

I have updated this with second approach. (Changes made to rest of tests, code wherever this is touched)
```
interface VideoSendingSource {
   attendee: Attendee
   // e.g.
   timestamp: number
}
```